src/map/entities/charentity.cpp

```diff
@@ -969,6 +969,23 @@ void CCharEntity::PostTick()
         pushPacket(new CCharAppearancePacket(this));
     }
 
+    // notify client containers are dirty and then no longer dirty
+    if (dirtyInventoryContainers.size() > 0)
+    {
+
+        // Notify client containers were dirty
+        // Note: Retail sends this in the same chunk as the inventory equip packets, but it doesnt seem to matter as long as it arrives
+        for (auto container: dirtyInventoryContainers)
```

Comment:
`const auto [container, dirty]`

Reply:
done (this is way better than using .first .second etc)

Reply:
Hmm. Clang doesn't like this:
```
 /home/runner/work/server/server/src/map/entities/charentity.cpp:978:25: error: loop variable '[container, dirty]' creates a copy from type 'std::pair<const CONTAINER_ID, bool> const' [-Werror,-Wrange-loop-construct]
        for (const auto [container, dirty]: dirtyInventoryContainers)
                        ^
/home/runner/work/server/server/src/map/entities/charentity.cpp:978:14: note: use reference type 'std::pair<const CONTAINER_ID, bool> const &' to prevent copying
        for (const auto [container, dirty]: dirtyInventoryContainers)
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                        &
```
Looks like it wants a ref. will play around with this

Reply:
It wanted: `const auto& [container, dirty]`